style.css

```diff
@@ -0,0 +1,1048 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Mochiy+Pop+One&family=Poppins:ital,wght@0,100;0,200;0,300;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+/* MOBILE VERSION */
+body {
+  background-color: #f8f8f8;
+  font-family: "Lato", sans-serif;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 0;
+  margin: 0;
```

Comment:
- [ ] As i said earlier, the aim is to have you have a very close replica to the design given. In that regards, kindly follow the color guides given for  most of the heading and other subheadings. 
-- Colors given include : `#272A31, #EC5242, #D3D3D3`

### view:
![image](https://github.com/maximoortelli/Capstone-Proyect/assets/50990925/b5d6881d-0d60-41c4-bb17-ad9996ac3157)

Reply:
Thank you very much for the suggestion, the changes have been made satisfactorily 👍😊